src/surrogates/src/InterpolationSurrogateIOASCII.C

```diff
@@ -49,7 +49,7 @@ namespace QUESO
                                                  int reading_rank )
   {
     // Root processor
-    unsigned int root = reading_rank;
+    int root = reading_rank;
```

Comment:
"to handle 2.2 billion rank machines.�

That�ll happen in a few months at the rate China is leveling up. :)


On Jun 20, 2016, at 3:10 PM, roystgnr <notifications@github.com> wrote:

> In src/surrogates/src/InterpolationSurrogateIOASCII.C:
> 
> > @@ -49,7 +49,7 @@ namespace QUESO
> >                                                   int reading_rank )
> >    {
> >      // Root processor
> > -    unsigned int root = reading_rank;
> > +    int root = reading_rank;
> 
> Yup. That should hold us for a decade or more; at least until we need to make the MPI-5 upgrade to handle 2.2 billion rank machines.
> 
> �
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>